CargoWebAsset.js

```diff
@@ -43,64 +38,42 @@ class CargoWebAsset extends Asset {
         }
     }
 
-    async install_nightly() {
-        let [stdout] = await exec( "rustup", ["show"] );
-        if( !stdout.includes( "nightly" ) ) {
+    static async install_nightly() {
+        const rustup_show = await exec( "rustup show" );
+        if( !rustup_show.stdout.includes( "nightly" ) ) {
             await pipeSpawn( "rustup", ["update"] );
             await pipeSpawn( "rustup", ["toolchain", "install", "nightly"] );
         }
     }
 
-    async install_cargo_web() {
-        if( cargo_web_command ) {
```

Comment:
I was a bit presumptuous in getting rid of the `cargo_web_command` class member. On the surface, it seemed like a means to cache the result of the cargo web install, but the version check seems like it would be negligible in terms of build times.